### ISSUE TITLE
Small mistakes in the DCGAN code 

### ISSUE THREAD

**Issue by lm17918** [aladdinpersson/Machine-Learning-Collection#23]
in the class Discriminator(nn.Module)  and Generator(nn.Module)
- the first and last Conv2d should have bias=False. and stride= 1, 
- remove comment from  nn.BatchNorm2d(out_channels).

**Comment by aladdinpersson** [comment]
Could you explain the first point?
